Resources/Audio/Items/Taperecorder/attributions.yml

```
@@ -0,0 +1,17 @@
+
+- files:
+    - "taperecorder_stop.ogg"
+  license: "CC-BY-4.0"
+  copyright: "Taken from Pressing Stop on An Old Tape Machine by djlprojects. Converted from Mp3 to Ogg."
+  source: "https://freesound.org/people/djlprojects/sounds/392889/"
+
+- files:
+    - "taperecorder_rewind.ogg"
+  license: "CC-BY-NC-4.0"
+  copyright: "Taken from CassetteRewind.flac by acclivity. Converted from Flac to Ogg."
+  source: "https://freesound.org/people/acclivity/sounds/23393/"
```

yamllint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)